lib/l10n/l10n.dart

```diff
@@ -313,3 +313,59 @@ extension L10nProfileTabExtension on ProfileTab {
     };
   }
 }
+
+/// Extension adding [int] formatting as a human-readable localized bytes.
+extension L10nSizeInBytesExtension on int? {
```

Comment:
@Gieted, please, check the latest commit I made. If you have any questions, be sure to leave. If everything's ok, then it's LGTM.

Reply:
Yep, your changes look good to me too